engine/src/vs_math.h

```diff
@@ -26,6 +26,7 @@
 #include "macosx_math.h"
 #include <math.h>
```

Comment:
Overall, this looks excellent. Absolutely outstanding! :tada:

Just one, relatively minor point: Now that we're including <cmath>, we probably don't need <math.h> any more. :sunglasses: 

Loving all the progress, you guys!